code/modules/modular_computers/computers/item/tablet.dm

```diff
@@ -39,6 +39,11 @@
 		explode(usr, from_message_menu = TRUE)
 		return
 
+/obj/item/modular_computer/tablet/examine(mob/user)
+	. = ..()
+	if(inserted_item)
+		. += "Ctrl-click to remove the [inserted_item.name]."
```

Comment:
Use a context screen tip instead, please! Here's an example of one, similar but not exactly what you would need:

```js
	AddElement( \
		/datum/element/contextual_screentip_bare_hands, \
		lmb_text = "Turn on", \
		rmb_text = "Turn off", \
	)
```

Reply:
I don't think you actually need backslashes here.